hyperspy/drawing/utils.py

```diff
@@ -343,6 +346,493 @@ def _set_spectrum_xlabel(spectrum, ax):
     ax.set_xlabel("%s (%s)" % (x_axis.name, x_axis.units))
 
 
+def plot_images(images,
+                cmap=None,
+                no_nans=False,
+                per_row=3,
+                label='auto',
+                labelwrap=30,
+                suptitle=None,
+                suptitle_fontsize=18,
+                colorbar='multi',
+                scalebar=None,
+                scalebar_color='white',
+                interp='nearest',
```

Comment:
Yes, you're correct that `interpolation` will work as a keyword. If this is done however, then the interpolation will default to `matplotlib`'s `rcParam` value. If this is okay, then I have no problem having interpolation as a keyword. Perhaps we could temporarily override the default to `'nearest'`, as shown [here](https://github.com/matplotlib/matplotlib/blob/master/doc/users/whats_new.rst#new-rc-parameter-functionality)?